karma.bootstrap.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+/* global*/
+
+console.info(`********* Karma Boostrap accessed`);	// eslint-disable-line no-console
```

Comment:
@jvercide-tc we may want to add more documentation as to how this works for other using this

Reply:
@cjardine-tc added some comments and ref link. also added some FAQs to wiki (https://github.com/tucows/donejs-carousel-plugin/wiki)